interface/forms/misc_billing_options/new.php

```diff
@@ -80,196 +81,214 @@
     $oemr_ui = new OemrUI($arrOeUiSettings);
     ?>
 </head>
-<body class="body_top">
-<div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?>">
-    <div class="row">
-        <div class="col-sm-12">
-            <?php echo  $oemr_ui->pageHeading() . "\r\n"; ?>
-        </div>
-    </div>
-    <div class="row">
-        <div class="col-sm-12">
+<body>
+    <div id="container_div" class="<?php echo attr($oemr_ui->oeContainer()); ?> mt-3">
```

Comment:
Can we not get rid of this $oemr_ui->oeContainer() hold over class for classes from 502? Same with the div above or whatever it is for collapse.

Reply:
What exactly does this code do?